### DISCUSSION TITLE
GitHub on iOS does not support Projects (beta)

### DISCUSSION THREAD

**Discussion by erikr** [community/community#5595] [Mobile]
GitHub on iPadOS or iOS does not support Projects (beta).

If I create a new issue on the mobile app, I cannot select a beta Project to associate with.

Only current (non-beta) Projects can be selected.

Is there an ETA for supporting beta Projects?

**Comment by candyho** [comment]
Hey👋you should be able to associate an issue or PR to a beta project at this point. Also, just want to inform you we've just shipped [view and edit projects(beta) custom fields]( https://github.blog/changelog/2022-05-09-view-and-edit-projectsbeta-custom-fields-on-github-mobile/) to both iOS and Android last week! You can now edit custom text, number, date, and single-select fields for issues and pull requests that belong to the new projects experience from within the triage sheet. Try it out and let us know what you think! Stay tuned for more updates!
